### ISSUE TITLE
030_FLAIR_BiasfieldCorrection.status not created during re-run

### ISSUE THREAD

**Issue by MauricePasternak** [ExploreASL/ExploreASL#123]
Minor compatibility issue with the GUI, which expects 030_FLAIR_BiasfieldCorrection.status to be created during a re-run if the user deletes all Structural Module status files. ExploreASL doesn't create this status file if the dataset is re-run:  

    WMH segmentation already existing, or FLAIR.nii missing, skipping FLAIR biasfield correction

Around the time that message is made, could 030_FLAIR_BiasfieldCorrection.status please be created [EDIT: only if WHM_SEGM.nii is detected] so that the GUI doesn't interpret the process as an error?

In other words, to re-create this:
1) Process the subject in TestDataSet as usual.
2) Delete all locked folders and .status files
3) Re-run
4) Observe that 030_FLAIR_BiasfieldCorrection.status is not made during the re-run.

@MichaelStritt something to keep in mind as you are testing the GUI, especially on the subject with a FLAIR image in TestDataSet

**Comment by MichaelStritt** [comment]
> In other words, to re-create this:
>     1. Process the subject in TestDataSet as usual.
>     2. Delete all locked folders and .status files
>     3. Re-run
>     4. Observe that 030_FLAIR_BiasfieldCorrection.status is not made during the re-run.
> 
> @MichaelStritt something to keep in mind as you are testing the GUI, especially on the subject with a FLAIR image in TestDataSet

Tested it with Henks recent fix a minute ago. Seems to work fine now.
